src/wh_she_common.c

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (C) 2024 wolfSSL Inc.
+ *
+ * This file is part of wolfHSM.
+ *
+ * wolfHSM is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfHSM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with wolfHSM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+ * src/wh_she_common.c
+ *
+ */
+/* System libraries */
+
+#include <stdint.h>
+#include <stdlib.h>  /* For NULL */
+#include <string.h>  /* For memset, memcpy */
+
+/* TODO replace with our own to host function */
+#include <arpa/inet.h>
```

Comment:
We can't merge this until you fix this TODO. We can't have `arpa/inet.h` on any of our real targets